File: cursopoouber/python/card.py
from payment import Payment

class Card(Payment):
   number = int
   cvv = int
   expDate = str
   owner = str

  def __init__(self, number, cvv, expDate, owner):
    super().__init__()
    self.number = number
    self.cvv = cvv
    self.expDate = expDate
    self.owner = owner

File: cursopoouber/php/card.php
<?php
require_once('payment.php');

class Card extends Payment{
  public $number;
  public $cvv;
  public $expDate;
  public $owner;
  public function __construct($number, $cvv, $expDate, $owner){
    parent::__construct();
    $this->number = $number;
    $this->cvv = $cvv;
    $this->expDate = $expDate;
    $this->owner = $owner;    
  }
}